src/main/scala/com/github.sstone/amqp/ChannelOwner.scala

```diff
@@ -51,10 +60,9 @@ object ChannelOwner {
           def handleFlow(active: Boolean): Unit = listener ! HandleFlow(active)
         }))
       }
-      case request@Publish(exchange, routingKey, body, properties, mandatory, immediate) => {
+      case request@Publish(_, _, _, _, _, _) => {
```

Comment:
Can you show me a reference to this? I wanted to understand that, but the only thing I found everywhere was what I wrote, that it's for having the whole object when you need to decompose the object also, which makes only sense if you wanna filter on some of the values of it, like `case p@Person(_, "Denton")`
Maybe you're referencing to type erasure here, when doing something like this will match on the first for every kind of Set:
```scala
a match {
    case x: Set[Int] => print("Int")
    case x: Set[String] => print("String")
}```

Now theres no type parameter for the `Publish` class, so no need to decompose the object just for this.